index.js

```diff
@@ -16,3 +16,265 @@ function closeMenu() {
+
+  container.appendChild(card);
+}
+
+const ProjectPopup = (
+  id,
+  title,
+  company,
+  post,
+  year,
+  image,
+  technologies,
+  details,
+  liveVersion,
+  githubLink,
+) => {
+  let technologyList = '';
+  for (let i = 0; i < technologies.length; i += 1) {
+    technologyList += `<li class="card-tag">${technologies[i]}</li>`;
+  }
+
+  return `<div class="pop-head">
+  <div class="pop-head-container">
+    <h3 class="card-title">${title}</h3>
+    <div class="card-info">
+      <span class="info-active">${company}</span>
+      <img src="./images/dot.svg" alt="dot" />
+      <span>${post}</span>
+      <img src="./images/dot.svg" alt="dot" />
+      <span>${year}</span>
+    </div>
+  </div>
+  <button id="pop-btn">&times;</button>
+</div>
+<div class="pop-img">
+  <img class="card-img" src="${image}" />
+</div>
+<div class="pop-content">
+  <p class="card-descrip card-item-mg">
+    ${details}
+  </p>
+  <div class="pop-footer">
+    <ul class="card-tags card-item-mg">
+      ${technologyList}
+    </ul>
+    <span class="span"></span>
+    <div class="pop-actions">
+      <button class="pop-button" onclick="window.open('${liveVersion}', '_blank')">
+        <span>See live</span>
+        <img src="images/live-pop.svg" />
+      </button>
+      <button class="pop-button" onclick="window.open('${githubLink}', '_blank')">
+        <span>See source</span>
+        <img src="images/github2.svg" />
+      </button>
+    </div>
+  </div>
+</div>`;
+};
+
+projects.forEach((aProject) => {
+  const btnShowProject = document.getElementById(`card-${aProject.id}`);
+  btnShowProject.addEventListener('click', () => {
+    const thePopupCardShows = document.getElementById('popupCard');
+    const htmlToInsert = ProjectPopup(
+      aProject.id,
+      aProject.title,
+      aProject.company,
+      aProject.post,
+      aProject.year,
+      aProject.image,
+      aProject.technologies,
+      aProject.details,
+      aProject.githubLink,
+      aProject.liveVersion,
+    );
+    thePopupCardShows.innerHTML = htmlToInsert;
+
+    const overly = document.getElementById('overly');
+    overly.classList.toggle('hidden');
+    thePopupCardShows.classList.toggle('popup-hidden');
+
+    const closePopBtn = document.getElementById('pop-btn');
+
+    closePopBtn.addEventListener('click', () => {
+      const thePopupCardShows = document.getElementById('popupCard');
+      thePopupCardShows.innerHTML = '';
+      overly.classList.toggle('hidden');
+      thePopupCardShows.classList.toggle('popup-hidden');
+    });
+  });
+});
+// form validation
+
+const form = document.getElementById('form');
+const email = document.getElementById('email_address');
+const error = document.getElementById('error-msg');
+
+error.style.display = 'none';
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    error.style.display = 'block';
+    error.innerHTML = 'invalid email!';
```

Comment:
- [ ] [OPTIONAL] I think you can use an error message like "Your email should be not contain uppercase" and the red color would be better as warning message